ui/job-view/headerbars/ActiveFilters.jsx

```diff
@@ -87,17 +85,17 @@ export default class ActiveFilters extends React.Component {
           <span
             className="pointable"
             title="Clear all of these filters"
-            onClick={this.thJobFilters.clearAllFilters}
+            onClick={filterModel.clearNonStatusFilters}
           ><i className="fa fa-times-circle" /> </span>
           <span className="active-filters-title">
             <b>Active Filters</b>
           </span>
           {filterBarFilters.map(filter => (
-            <span className="filtersbar-filter" key={`${filter.key}${filter.value}`}>
+            <span className="filtersbar-filter" key={`${filter.field}${filter.value}`}>
```

Comment:
``filter.key`` was always the same value as ``filter.field`` so I got rid of ``key``.